diesel/src/sqlite/expression/helper_types.rs

```diff
@@ -7,143 +7,4 @@ pub type Is<Lhs, Rhs> = Grouped<super::operators::Is<Lhs, AsExpr<Rhs, Lhs>>>;
-#[cfg(feature = "sqlite")]
-pub type jsonb_group_object<N, V> =
-    super::functions::jsonb_group_object<SqlTypeOf<N>, SqlTypeOf<V>, N, V>;
-
-/// Return type of [`json_array_0()`](super::functions::json_array_0())
-#[allow(non_camel_case_types)]
-#[cfg(feature = "sqlite")]
-pub type json_array_0 = super::functions::json_array_0;
-
-/// Return type of [`json_array_1(value_1)`](super::functions::json_array_1())
-#[allow(non_camel_case_types)]
-#[cfg(feature = "sqlite")]
-pub type json_array_1<V1> = super::functions::json_array_1<SqlTypeOf<V1>, V1>;
-
-/// Return type of [`json_array_2(value_1, value_2)`](super::functions::json_array_2())
-#[allow(non_camel_case_types)]
-#[cfg(feature = "sqlite")]
-pub type json_array_2<V1, V2> =
-    super::functions::json_array_2<SqlTypeOf<V1>, SqlTypeOf<V2>, V1, V2>;
-
-/// Return type of [`jsonb_array_0()`](super::functions::jsonb_array_0())
-#[allow(non_camel_case_types)]
-#[cfg(feature = "sqlite")]
-pub type jsonb_array_0 = super::functions::jsonb_array_0;
-
-/// Return type of [`jsonb_array_1(value_1)`](super::functions::jsonb_array_1())
-#[allow(non_camel_case_types)]
-#[cfg(feature = "sqlite")]
-pub type jsonb_array_1<V1> = super::functions::jsonb_array_1<SqlTypeOf<V1>, V1>;
-
-/// Return type of [`jsonb_array_2(value_1, value_2)`](super::functions::jsonb_array_2())
-#[allow(non_camel_case_types)]
-#[cfg(feature = "sqlite")]
-pub type jsonb_array_2<V1, V2> =
-    super::functions::jsonb_array_2<SqlTypeOf<V1>, SqlTypeOf<V2>, V1, V2>;
-
-/// Return type of [`json_remove_0(json)`](super::functions::json_remove_0())
-#[allow(non_camel_case_types)]
-#[cfg(feature = "sqlite")]
-pub type json_remove_0<J> = super::functions::json_remove_0<SqlTypeOf<J>, J>;
-
-/// Return type of [`json_remove_1(json, path_1)`](super::functions::json_remove_1())
-#[allow(non_camel_case_types)]
-#[cfg(feature = "sqlite")]
-pub type json_remove_1<J, P1> = super::functions::json_remove_1<SqlTypeOf<J>, J, P1>;
-
-/// Return type of [`json_remove_2(json, path_1, path_2)`](super::functions::json_remove_2())
-#[allow(non_camel_case_types)]
-#[cfg(feature = "sqlite")]
-pub type json_remove_2<J, P1, P2> = super::functions::json_remove_2<SqlTypeOf<J>, J, P1, P2>;
-
-/// Return type of [`jsonb_remove_0(json)`](super::functions::jsonb_remove_0())
-#[allow(non_camel_case_types)]
-#[cfg(feature = "sqlite")]
-pub type jsonb_remove_0<J> = super::functions::jsonb_remove_0<SqlTypeOf<J>, J>;
-
-/// Return type of [`jsonb_remove_1(json, path_1)`](super::functions::jsonb_remove_1())
-#[allow(non_camel_case_types)]
-#[cfg(feature = "sqlite")]
-pub type jsonb_remove_1<J, P1> = super::functions::jsonb_remove_1<SqlTypeOf<J>, J, P1>;
-
-/// Return type of [`jsonb_remove_2(json, path_1, path_2)`](super::functions::jsonb_remove_2())
-#[allow(non_camel_case_types)]
-#[cfg(feature = "sqlite")]
-pub type jsonb_remove_2<J, P1, P2> = super::functions::jsonb_remove_2<SqlTypeOf<J>, J, P1, P2>;
+pub use super::functions::return_type_helpers::*;
```

Comment:
Somehow that module shows up in the documentation: 

![image](https://github.com/user-attachments/assets/2f7b5fa9-dd71-4d65-b75c-91bb489f8cf8)

That's in the docs of `diesel::sqlite::expression::dsl`

(There is also a list of exported helper types below, so it mostly works but not 100%)